### PR TITLE
[2.0] New: Add polymer lint to lint script

### PR DESCRIPTION
**Pull request by stramel**
Seems like all lint errors were fixed by @justinfagnani. It would be a shame not to go ahead and lint

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "eslint-plugin-html": "^2.0.0"
   },
   "scripts": {
-    "lint": "npm run lint:javascript",
+    "lint": "npm run lint:javascript && polymer lint src/*.html",
```
Comment (abdonrd) [inline package.json]: Normally we only use `polymer lint`.

Comment (stramel) [inline package.json]: @abdonrd So, `polymer lint` will lint all the files including the build directory.

Steps to Reproduce:

1. `npm i -g polymer-cli`
1. `polymer build`
1. `polymer lint`

Comment (abdonrd) [inline package.json]: Can we do something like ESLint ignore path?
`eslint . --ignore-path .gitignore`

cc @rictic

Comment (stramel) [inline package.json]: Currently, this isn't implemented, although probably would be a great feature!

Comment (stramel) [inline package.json]: @abdonrd Is this holding this PR from moving forward?

Comment (abdonrd) [inline package.json]: While waiting for https://github.com/Polymer/polymer-linter/issues/88...

As Travis do the lint first, remove the to `src/*.html` and I'll merge it!

**Comment by stramel** [comment]
@abdonrd Updated!

**Comment by abdonrd** [comment]
Thanks for the contribution @stramel! 👏

**Pull request merged**